src/test/java/org/checkerframework/specimin/SpeciminTestExecutor.java

```diff
@@ -90,6 +90,8 @@ public static void runTest(
       builder.command(
           "diff",
           "-qr",
+          "-w",
+          "-B",
```

Comment:
Why do you need `-w` and `-B`? I thought `-w` implied `-B`, but I might be misremembering the various `diff` options.

Reply:
When I played around with these option, I noticed that "-w" will ignore white spaces but does not ignore blank lines, and as `diff` compares line by line, we will also need "-B" to ignore blank lines.

Reply:
I've probably just misremembered - I should have checked myself. Sorry for the confusion.